configure.ac

```diff
@@ -1489,6 +1489,90 @@ AC_MSG_NOTICE([checking package dependencies])
 
 AC_PATH_PROG([PKG_CONFIG],[pkg-config])
 
+#
+# Chip over Ble over Bluez Peripheral support
+#
+
+# Only a concern when the BLE layer is enabled.
+
+CONFIG_BLE_PLATFORM_BLUEZ=0
+enable_woble_bluez="no"
```

Comment:
woble->chipoble?

Reply:
Hi, Rob, 

thanks for catching up the typos, I will fix them!

Best wishes
Yunhan

Reply:
Done, thanks